kong/conf_loader/init.lua

```diff
@@ -658,8 +658,13 @@ local CONF_INFERENCES = {
   untrusted_lua_sandbox_requires = { typ = "array" },
   untrusted_lua_sandbox_environment = { typ = "array" },
 
+  legacy_worker_events = { typ = "boolean" },
+
   lmdb_environment_path = { typ = "string" },
   lmdb_map_size = { typ = "string" },
+
+  opentelemetry_tracing = { typ = "array" },
```

Comment:
It seems that `opentelemetry_tracing_*` was included by mistake

Reply:
I did it just for solving the merge conflict.

Reply:
#8890 fix it.